proto/beacon/rpc/v1/services.proto

```diff
@@ -22,23 +22,21 @@ service ProposerService {
 }
 
 service ValidatorService {
-    rpc ValidatorShardID(PublicKey) returns (ShardIDResponse);
-    rpc ValidatorIndex(PublicKey) returns (IndexResponse);
-    rpc ValidatorSlot(PublicKey) returns (SlotResponse);
```

Comment:
Why remove the previous methods?

Reply:
We won't need them - it doesn't make sense for a validator to call those on-demand. It makes more sense for a beacon node to stream them because if a validator requests its index while the node is in the middle of calculating a state transition, then the wrong response might be sent. That's why we decided on doing streaming from the server as an alternative.